jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MimeTypes.java

```diff
@@ -653,9 +653,9 @@ public String addAssumed(String mimeType, String encoding)
          * @param mimeProperties The property file to use.
          * @see #addMimeTypes(Resource)
          * @see #addMimeMapping(String, String)
-         * @throws RuntimeIOException if there is an {@link IOException}
+         * @throws UncheckedIOException if there is an {@link IOException}
          */
-        public void setMimeTypes(Resource mimeProperties) throws RuntimeIOException
+        public void setMimeTypes(Resource mimeProperties) throws UncheckedIOException
```

Comment:
Do we still need to declare runtime exceptions here?
Couldn't this just be ...

```suggestion
        public void setMimeTypes(Resource mimeProperties)
```

... now?
Ditto for other similar cases.

Reply:
There is no NEED to declare, but for documentation purposes it is good practise to declare